### ISSUE TITLE
Sort through the existing NixOS repositories to precise their status

### ISSUE THREAD

**Issue by thufschmitt** [NixOS/teams-collaboration#1]
Cleaning-up the list of repositories under https://github.com/orgs/NixOS/repositories should give a pretty good first idea of what's going on in the ecosystem. I've collected the list [here](https://pad.lassul.us/_O7BJW5mR56pM41u3nW3ZQ) with a first estimation of their status. I propose that we:

1. Confirm the statuses with the relevant people
2. Archive the dead things
3. Explicit the maintenance of the living things where it's unclear.

For 3., the [nix-community](https://github.com/nix-community) model of specifying the maintainers directly in the project description seems pretty good (except that we probably should make it a GitHub team as the official NixOS projects should have much more of a shared ownership)

**Comment by thufschmitt** [comment]
I've just opened [an issue](https://github.com/NixOS/nixops/issues/1574) in NixOps suggesting to sunset it one way or another

**Comment by thufschmitt** [comment]
Update on that: I have [a script](https://gist.github.com/thufschmitt/6ce73ec2ac7f38be3ee748ab16ba3aa5) to go over all the repositories where someone acknowledged the archival or no one reacted.

@NixOS/admins, can someone either give me temporary admin rights or run that script (after careful review, of course) to act on it?

**Comment by infinisil** [comment]
@thufschmitt That team name is unfortunate, it's not what you think it is :smile:

**Comment by vcunat** [comment]
github.com/NixOS admins?  Off the top of my head, @zimbatm, @domenkozar ?

**Comment by thufschmitt** [comment]
Thanks @infinisil @vcunat (and I don't think it's the first time I get bitten by this…)

**Comment by thufschmitt** [comment]
Just ran the script, this should do a good bit of clean-up. Thanks @zimbatm for the access

**Comment by DanielSidhion** [comment]
Hi @thufschmitt - I think we should reopen https://github.com/NixOS/snapd-nix-base because it's still needed, even thought I expect changes to it will be very rare. Some context:

I'm working on updating some content on the Nixpkgs manual and just came across `pkgs.makeSnap`. [This is the source](https://github.com/NixOS/nixpkgs/blob/1dfd9a62f32d09b918744985efb1b18e542f30d9/pkgs/build-support/snap/make-snap.nix) for it.

That source mentions [in these lines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/snap/make-snap.nix#L30-L32) that it declares a base snap, which is built from https://github.com/NixOS/snapd-nix-base

I think as long as `makeSnap` has that implementation, we should keep that repository unarchived.

**Comment by thufschmitt** [comment]
Thanks for the info @DanielSidhion .

There's apparently an official empty snap on the repository: https://snapcraft.io/bare, do you want to open a PR to Nixpkgs to replace `nix-base` with `bare` as the base? That way we don't have to depend on our own unmainained version

EDIT: I didn't see that ours had a `/nix` at the root, we might want to check that it doesn't break to not have it

**Comment by DanielSidhion** [comment]
I spent too many hours trying to better understand the snap specs and environment and trying to come up with a patch that works, but I think that given the current snap specs, the current `makeSnap` can't be patched to use that official empty snap (or any other empty snap, including the existing `nix-base`) without other heavy modifications to it.

That is currently documented in https://github.com/NixOS/nixpkgs/issues/293376

Given the current state of `makeSnap`, I think it's ok to keep that other repo archived. Any efforts made to fix `makeSnap` can also include moving to a different empty base snap.

**Comment by thufschmitt** [comment]
Sad thing to hear, but thanks for the investigation @DanielSidhion !

I'll keep it archived until/unless someone offers to fix the snap toolchain

**Comment by zowoq** [comment]
The `nixos-weekly` repo is pinned, now it has been archived should switch it out for another repo.

![nixos-weekly](https://github.com/NixOS/teams-collaboration/assets/59103226/80702621-ffdf-44d6-8c9d-c3518e37b0ad)

**Comment by thufschmitt** [comment]
> The `nixos-weekly` repo is pinned, now it has been archived should switch it out for another repo.

Good catch. I've replaced it by patchelf (most starred repo that wasn't pinned). Happy for suggestions if folks prefer another one
